.bruno/400-Wrong-Eori.bru

```diff
@@ -18,6 +18,6 @@ body:json {
   {
     "validityDate": "{{date}}",
     "authType": "UKNW",
-    "eoris" : ["JsdbuwnNN%$400"]
+    "eoris" : ["ABCD000000000200"]
```

Comment:
I would rather change the expected eori in the stub since "ABCD000000000200" represents a good eori and shouldn't trigger an error.

Though to be honest it wouldn't make any difference in terms of functionality so feel free to ignore this comment